src/vm/eventpipe.cpp

```diff
@@ -392,7 +389,7 @@ EventPipeSession *EventPipe::GetSession(EventPipeSessionID id)
     {
         CrstHolder _crst(GetLock());
 
-        if (!s_tracingInitialized)
+        if (s_state < EventPipeState::Initialized)
```

Comment:
```suggestion
        if (s_state == EventPipeState::NotInitialized)
```
Nit: if you wanted to be consistent with the other checks